rs/protobuf/src/gen/p2p/registry.crypto.v1.rs

```diff
@@ -0,0 +1,210 @@
+/// A list of subnets that can sign with this ECDSA key.
```

Comment:
Since we now have 2k lines of duplicate generated protobuf code it might indeed be better to put the type next to the consensus protobufs.